hs/src/Reach/Backend/JS.hs

```diff
@@ -698,8 +698,13 @@ jsConnsExp names = "export const _Connectors" <+> "=" <+> jsObject connMap <> se
   where
     connMap = M.fromList [(name, "_" <> pretty name) | name <- names]
 
+jsExportValue :: DLExportValue -> App Doc
+jsExportValue = \case
+  DLEV_Arg a -> jsArg a
+  DLEV_LArg a -> jsLargeArg a
```

Comment:
Functions will have to `jsProtect` their arguments based on the types of the corresponding `DLVar`